src/bin/opensmt.cc

```diff
@@ -48,6 +48,13 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 namespace opensmt {
 
+
+void printHelp( );
+
+void printVersion( );
+
+void parseCMDLine( int argc, char * argv[ ] );
```

Comment:
Can you fix the formatting?

```suggestion
void printHelp();

void printVersion();

void parseCMDLine( nt argc, char * argv[]);
```

Reply:
Fixed